runtime/vm/jvminit.c

```diff
@@ -1823,19 +1824,41 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				IDATA argIndexCompactOnIdleDisable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGCOMPACTONIDLEDISABLE, NULL);
 				IDATA argIndexIgnoreUnrecognizedOptionsEnable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSENABLE, NULL);
 				IDATA argIndexIgnoreUnrecognizedOptionsDisable = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXIDLETUNINGIGNOREUNRECOGNIZEDOPTIONSDISABLE, NULL);
+				BOOLEAN enableGcOnIdle = FALSE;
+				BOOLEAN inContainer = FALSE;
+				int32_t errorCode = 0;
+				
+				inContainer = omrsysinfo_is_running_in_container(&errorCode);
+				inContainer = ((0 == errorCode) && (TRUE == inContainer));
```

Comment:
This seems reversed. The return value is in32_t and zero for success. The parameter is a BOOLEAN set to TRUE when running in a container.

Reply:
@pshipton Yeah it looks reversed, but it is as per the api `omrsysinfo_is_running_in_container`. See https://github.com/eclipse/omr/blob/85e2a6fd463e232126565956ca30cfe3340b483e/port/common/omrsysinfo.c#L962

Reply:
Guess the api changed at some point and I had an old copy checked out.